src/databases/mssql.database.ts

```
@@ -0,0 +1,777 @@
+			case MSSQLColumnType.NTEXT:
+			case MSSQLColumnType.NCHAR:
+			case MSSQLColumnType.NVARCHAR:
+				return DatabaseColumnType.STRING
+			case MSSQLColumnType.DATE:
+			case MSSQLColumnType.DATETIME:
+			case MSSQLColumnType.DATETIME2:
+			case MSSQLColumnType.SMALLDATETIME:
+			case MSSQLColumnType.DATETIMEOFFSET:
+			case MSSQLColumnType.TIME:
+				return DatabaseColumnType.DATE
+			case MSSQLColumnType.SQL_VARIANT:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)